packages/trip-form/src/__mocks__/modes-en.ts

```diff
@@ -194,4 +196,19 @@ const commonModes = {
   ]
 };
 
+// This variant of the data adds an option with a company as the first
+// option in the "CAR" category that has mixed modes ("CAR_PARK", "CAR_HAIL")
+const modesWithCompanyFirstMixedCategory = cloneDeep(commonModes);
+modesWithCompanyFirstMixedCategory.categories.forEach(c => {
```

Comment:
A bit of a nitpick, but I'm a little confused as to why we're using `forEach` here instead of `find` or similar. I guess this might be fewer lines of code so I think a comment at the very least explaining why `forEach` is used would be helpful!

Reply:
Ah, yes `find` is better, will change

Reply:
fixed!